index.ts

```diff
@@ -11,6 +11,13 @@ import { Client, estypes, ClientOptions } from "@elastic/elasticsearch";
 import { StdioServerTransport } from "@modelcontextprotocol/sdk/server/stdio.js";
 import fs from "fs";
 
+// Product metadata, used to generate the request User-Agent header and 
+// passed to the McpServer constructor.
+const product = {
+  name: "elasticsearch-mcp-server",
+  version: "0.1.1",
+};
```

Comment:
Quick improvement so you don't have to remember to modify this file before every release.

```suggestion
// Product metadata, used to generate the request User-Agent header and 
// passed to the McpServer constructor.
const { version } = JSON.parse(fs.readFileSync('package.json'))
const product = {
  name: "elasticsearch-mcp-server",
  version,
};
```

Reply:
Good call, I actually wondered about this while I was writing it :)